src/status_im/navigation/screens.cljs

```diff
@@ -136,544 +137,807 @@
+   :metrics   {:track? true}
+   :component identifiers/view
+   :options   {:theme              :dark
+               :layout             options/onboarding-transparent-layout
+               :animations         transitions/push-animations-for-transparent-background
+               :popGesture         false
+               :hardwareBackButton {:dismissModalOnPress false
+                                    :popStackOnPress     false}}})
+
+(def onboarding-sign-in-intro
+  {:name      :screen/onboarding.sign-in-intro
+   :metrics   {:track? true}
+   :options   {:layout                 options/onboarding-transparent-layout
+               :animations             (merge
+                                        transitions/sign-in-modal-animations
+                                        transitions/push-animations-for-transparent-background)
+               :modalPresentationStyle :overCurrentContext}
+   :component sign-in/animated-view})
+
+;; TODO(@seanstrom): Remove this definition if it is no longer needed
+(def onboarding-sign-in
+  {:name      :screen/onboarding.sign-in
+   :metrics   {:track? true}
+   :options   {:theme                  :dark
+               :modalPresentationStyle :overCurrentContext
+               :layout                 options/onboarding-layout}
+   :component sign-in/view})
+
+;; NOTE(@seanstrom):
+;; This screen seems to be accessible from settings when syncing.
+;; Should we consider this an onboarding screen?
+(def onboarding-syncing-progress
+  {:name      :screen/onboarding.syncing-progress
+   :metrics   {:track?   true
+               :alias-id :onboarding.syncing-devices}
+   :options   (assoc options/dark-screen
+                     :popGesture
+                     false)
+   :component syncing-devices/view})
+
+(def onboarding-syncing-progress-intro
+  {:name      :screen/onboarding.syncing-progress-intro
+   :metrics   {:track? true}
+   :options   {:theme      :dark
+               :layout     options/onboarding-transparent-layout
+               :animations transitions/push-animations-for-transparent-background
+               :popGesture false}
+   :component syncing-devices/view-onboarding})
+
+(def onboarding-syncing-results
+  {:name      :screen/onboarding.syncing-results
+   :metrics   {:track? true}
+   :options   {:theme :dark}
+   :component syncing-results/view})
+
+(defn onboarding-screens
+  []
+  [onboarding-intro
+   onboarding-new-to-status
+   onboarding-sync-or-recover-profile
+   onboarding-create-profile
+   onboarding-create-profile-password
+   onboarding-enable-biometrics
+   onboarding-generating-keys
+   onboarding-preparing-status
+   onboarding-entering-seed-phrase
+   onboarding-enable-notifications
+   onboarding-identifiers
+   onboarding-sign-in-intro
+   onboarding-sign-in
+   onboarding-syncing-progress
+   onboarding-syncing-progress-intro
+   onboarding-syncing-results])
+
 (defn screens
   []
   (concat
    (old-screens/screens)
+   (chat-screens)
```

Comment:
Why do we need to `defn` and then call it later ? Why not just `def` and screens can just be:

```clj
(def chat-screens
  ...)

(concat 
  old-screens/screens 
  chat-screens ...)
```

Reply:
Yeah I think it's possible that they can all be turned into defs, at the time I assumed there was a reason for `screens` being a function, but maybe that's for legacy reasons.

cc @ilmotta

Reply:
As far as I remember, one of the reasons to choose functions is that we sometimes may prefer to defer execution to be triggered at a later stage, so that we impact less the already slow(ish) startup of the app (in some devices). But this idea doesn't seem to make much sense for this code, so probably unrelated.

Might be just personal preference. I've seen devs who prefer to use functions, give names to them, and then treat the vars as global state, which means they might prefer to put as little code as possible in the var definition directly. A matter of taste maybe.